rabbitmq/src/urabbitmq/client_settings.cpp

```diff
@@ -45,6 +46,46 @@ AuthSettings Parse(const formats::json::Value& doc,
   auth.password = doc["password"].As<std::string>();
   auth.vhost = doc["vhost"].As<std::string>();
 
+  TlsSettings tls_settings;
+  const auto& client_cert_path = doc["tls"]["client-cert-path"];
+  const auto& client_key_path = doc["tls"]["client-key-path"];
+  if (client_cert_path.IsMissing() != client_key_path.IsMissing()) {
+    throw std::runtime_error(
+        "Either set both tls.client-cert-path and "
+        "tls.client-key-path options or none of "
+        "them");
+  }
+  if (!client_cert_path.IsMissing()) {
+    ClientCertSettings client_cert_settings;
+
+    const auto& client_cert_contents =
+        fs::blocking::ReadFileContents(client_cert_path.As<std::string>());
```

Comment:
This blocks the thread, and considering that this code is executed at components initialization, which is done in parallel, might lead to application start slowdown.
Is there a way to use [this overload](https://github.com/userver-framework/userver/blob/a2d8c5cd81737a1608aebe97fb0ba279127a5f32/core/include/userver/fs/read.hpp#L58) instead?